tests/phpunit/tests/multisite/cleanDirsizeCache.php

```diff
@@ -311,15 +311,16 @@ function test_5_5_transient_structure_compat() {
 		}
 
 		function _get_mock_5_5_dirsize_cache( $site_id ) {
+			$prefix = untrailingslashit( WP_CONTENT_DIR );
```

Comment:
would it be better to use `wp_upload_dir()['basedir']`, since that's what `get_space_used()` uses? i'm not sure if `WP_CONTENT_DIR` would match in all circumstances, since `_wp_upload_dir()` sets it in various ways.

_( that may be a dumb question, though, but brain is pretty fried at this point )_